README.md

```diff
@@ -0,0 +1,73 @@
+[![Build Status](https://travis-ci.org/internap/netman.svg?branch=master)](https://travis-ci.org/internap/redlock-fifo)
+[![PyPI version](https://badge.fury.io/py/redlock-fifo.svg)](http://badge.fury.io/py/redlock-fifo)
+
+Redlock-Fifo
+============
+
+Redlock-Fifo makes [Redlock-py](https://github.com/SPSCommerce/redlock-py) usable in a multi-threaded environment.
+
+The project make sure that the requester for a lock will have it according to the order it asked for access. It also 
+adds the ability to extends a lock without having to release the lock.
+
+Why would I use this ?
+----------------------
+The normal behavior of redlock-py is to `return false` on a existing lock. The mainstream approach is to 
+poll the lock until acquisition. This process has two downsides :
+
+1. It doesn't warranty that the first requester will be the first to get the lock.
```

Comment:
IMO warranty is not a verb so it can't be used likewise. Perhaps guarantee would be better...